components/meta-packages/build-essential/Makefile

```diff
@@ -17,6 +17,7 @@ include ../../../make-rules/shared-macros.mk
 
 COMPONENT_NAME=		build-essential
 COMPONENT_VERSION=	1.0
+COMPONENT_REVISION=	1
 COMPONENT_SUMMARY=	A meta-package that installs common development tools such as gcc
```

Comment:
No need to change the COMPONENT_REVISION as RELEASE_MAJOR has been changed to 2022 recently.

Reply:
Nevertheless we need a change in the Makefile in order to get jenkins buiding it.